### PR TITLE
feat(JAAS in SP): move JAAS implem to SP only

### PR DESCRIPTION
**Pull request by educhastenier**
Closes [RUNTIME-128](https://bonitasoft.atlassian.net/browse/RUNTIME-128)

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -54,6 +54,12 @@ default log level for `com.bonitasoft` and `org.bonitasoft` change from `WARNING
 From this release, upgrade from Community to Subscription is made [a lot easier](upgrade-from-community-to-a-subscription-edition.md#bonita-platform-upgrade):
 all the tricky configuration part is handled automatically by Bonita Runtime at server startup.
 
+#### JAAS authentication is now Enterprise
+
+From version 2021.2, JAAS Authentication is only available in Enterprise edition.  
+If you are updating from a 2021.1 or earlier version, in Community edition, you will need to [change your configuration](BonitaBPM_platform_setup.md) file `authenticationManager-config.properties`
```
Comment (educhastenier) [inline md/release-notes.md:60]: @abirembaut Do we also need to remove file `jaas-standard.cfg` under tomcat/server/conf ?

Comment (abirembaut) [reply md/release-notes.md:60]: @educhastenier It is already only in the subscription edition

**Review by abirembaut — APPROVED** [review]
LGTM

File: md/release-notes.md [hunk 2]
```diff
@@ -54,6 +54,12 @@ default log level for `com.bonitasoft` and `org.bonitasoft` change from `WARNING
...
+and remove the JAAS configuration to use another Configuration Manager (an implementation of your own, for instance).
```
Comment (abirembaut) [inline md/release-notes.md]:
```suggestion
and remove the JAAS configuration to use another Authentication Manager (an implementation of your own, for instance).
```

**Pull request merged**